src/Microsoft.AspNetCore.Mvc.Core/Controllers/IControllerArgumentBinder.cs

```diff
@@ -9,15 +9,20 @@ namespace Microsoft.AspNetCore.Mvc.Controllers
     /// <summary>
     /// Provides a dictionary of action arguments.
     /// </summary>
-    public interface IControllerActionArgumentBinder
+    public interface IControllerArgumentBinder
```

Comment:
But now it's a `public` API change. Might be better to fix the filename instead. (Admittedly this isn't a prominent namespace nor an in-your-face method.)